drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/AbstractSelectedColumnCommand.java

```diff
@@ -48,10 +50,17 @@
  */
 public abstract class AbstractSelectedColumnCommand extends AbstractScenarioSimulationCommand {
 
+    protected FactMappingValueType factMappingValueType = FactMappingValueType.RAW;
+
     public AbstractSelectedColumnCommand() {
         super(true);
     }
 
+    public AbstractSelectedColumnCommand(FactMappingValueType factMappingValueType) {
```

Comment:
@yesamer 
Pls remove that - having two constructors invoked for the same kind ("AbstractSelectedColumnCommand") of command in this situation is not justified by the logic behind